web/cypress/integration/route/can-skip-upstream-when-select-service-id.spec.js

```diff
@@ -18,19 +18,25 @@
 
 context('Can select service_id skip upstream in route', () => {
   const data = {
-    upstreamName: 'test_upstream',
-    serviceName: 'test_service',
-    routeName: 'test_route',
+    upstream_name: 'test_upstream',
```

Comment:
Why use UnderScoreCase？

Reply:
it's called snake_case and the API will return such a case, to reduce transformation, we allow to use that case.